docs/integrations/flutter-specific-workflow-steps/flutter-install.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Flutter Install
+metaTitle: Flutter Install
+metaDescription: Flutter Install
+sidebar_position: 1
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Flutter Install
+
+This workflow step installs the specified Flutter SDK to run the Flutter CLI for subsequent, Analyze, Build and Test operations. The Flutter version can be specified in [Configuration](https://docs.appcircle.io/build/building-flutter-applications/#build-configuration-for-flutter-applications)
+
+:::info
+All Flutter versions and detailed information can be found in the [Flutter repository](https://github.com/flutter/flutter).
+:::
+
+### Prerequisites
+
+In order for this step to work properly and not negatively affect other steps connected to the [**Flutter SDK**](https://github.com/flutter/flutter) such as [**Flutter Build for iOS**](https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-build-for-ios) or [**Flutter Build for Android**](https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-build-for-android), please use it before the following steps.
```

Comment:
Is there a reason to repeat here exactly the same dependencies that we wrote in table? If not, it might be better to delete it for simplicity.